### ISSUE TITLE
Google Analytics

### ISSUE THREAD

**Issue by jvolima** [LukeberryPi/phived#51]
I suggest adding google analytics to the application so that you can access your site's metrics, such as: number of hits, understanding user behavior, improving the site's search engine, among other advantages.

**Comment by LukeberryPi** [comment]
heyo, couple q's:

- how much code does this involve?
- what is the main insight you'd be interested in knowing?
- i remember tinkering with Google Analytics and getting the number of all-time accesses to phived (around 43k a week ago), but i did it all through their website (no code involved).

tell me some more and we see how we'll go about it :)

**Comment by jvolima** [comment]
I apologize for the delay in responding.

First, you just need to add a script in index.html.

Second, Google Analytics provides data for you to analyze, such as: 
1. Traffic Sources: Google Analytics can show you where your website traffic comes from, including search engines, social media, direct traffic, and more.
2. Audience Demographics: You can learn about the age, location, interests and behaviors of your website visitors.

In short, google analytics can help you make decisions to increase the number of users on your site, improve the user experience, among others.

**Comment by LukeberryPi** [comment]
![image](https://user-images.githubusercontent.com/98068247/235798638-28dbdf08-2e18-4871-afb2-21103519ac2f.png)

i had done this already, let me know of any insights you may wish :)

closing this :)
